src/web/screens/dashboard.tsx

```diff
@@ -31,7 +32,7 @@ export const StandardRoutes: PortalRoute[] = [
   TermsOfServiceRoute,
 ];
 
-export const AdminRoutes: PortalRoute[] = [ManageParticipantsRoute];
+export const AdminRoutes: PortalRoute[] = [ManageParticipantsRoute, KeyPairRoute];
```

Comment:
I think KeyPair is a client-facing feature, it is not an admin route?

Reply:
I'm currently just hiding it here until it's ready for clients.  There's a lot of more work to be done on it.

Reply:
Makes sense, though maybe a good idea to add a comment in code to that effect?

Reply:
added a note.  thanks.